test/model_tests/importing.test.js

```diff
@@ -270,4 +270,37 @@ describe('importing', () => {
       expect(this.application.itemManager.tags.length).to.equal(1);
       expect(this.application.findItem(note.uuid).deleted).to.be.false;
     });
+
+    it('should import data from encrypted payload', async function () {
```

Comment:
This test should be called "should import data from 003 encrypted payload" and we should have another test called "should import data from 004 encrypted payload"